app/test_main.py

```diff
@@ -1,3 +1,91 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
```

Comment:
using classes for tests is not mandatory in pytest. You can write standalone test functions